components/chart/src/selection-area/selection-area.ts

```diff
@@ -803,17 +809,32 @@ export class DtChartSelectionArea implements AfterContentInit, OnDestroy {
 
     // handling of the overlay for the range
     if (this._chart._range) {
+      merge(
+        getElementRefStream<HTMLDivElement>(
```

Comment:
please create this stream and assign to a const you create the same stream some lines below